litmus-portal/frontend/src/components/SettingsComponents/AccountsTab/PersonalDetails/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { Button, Modal, Typography } from '@material-ui/core';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { UserData } from '../../../../models/user';
+import { RootState } from '../../../../redux/reducers';
+import UserDetails from '../../UserManagementTab/CreateUser/UserDetails';
+import useStyles from './styles';
+
+interface personaData {
+  email: string;
+  userName: string;
+  fullName: string;
+}
+
+// Displays the personals details on the "accounts" tab
+const PersonalDetails: React.FC = () => {
+  const classes = useStyles();
+
+  const userData: UserData = useSelector((state: RootState) => state.userData);
+
+  const { name, email, username } = userData;
+  // console.log("user-"+ username);
+  const [personaData, setPersonaData] = React.useState<personaData>({
+    email,
+    userName: username,
+    fullName: name,
+  });
+
+  // For closing and opening of the modal
+  const [open, setOpen] = React.useState(false);
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <form>
+        <UserDetails
+          emailIsDisabled={false}
+          nameIsDisabled={false}
+          nameValue={personaData.fullName}
+          usernameIsDisabled
+          handleNameChange={(e) => {
+            setPersonaData({
+              fullName: e.target.value,
+              userName: personaData.userName,
+              email: personaData.email,
```

Comment:
Not a necessary change but if you keep the handlers outside and apply the logic there in the function and just put a call to that function here it would be a better approach. Just like `handleClose`